Sources/Commands/SwiftTestTool.swift

```diff
@@ -176,7 +176,7 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
     }
 
     override func runImpl() throws {
-        
+        let fileManager = Basic.localFileSystem
```

Comment:
This assignment is not really required, you can directly use `localFileSystem` at the call site.

Reply:
done